rust/pact_ffi/src/mock_server/mod.rs

```diff
@@ -419,7 +421,27 @@ pub extern fn pactffi_mock_server_mismatches(mock_server_port: i32) -> *mut c_ch
 #[no_mangle]
 pub extern fn pactffi_cleanup_mock_server(mock_server_port: i32) -> bool {
   let result = catch_unwind(|| {
-    pact_mock_server::shutdown_mock_server(mock_server_port)
+    let id = pact_mock_server::find_mock_server_by_port(mock_server_port as u16, &|_, id, mock_server| {
+      let interactions = match mock_server {
+        Either::Left(ms) => {
+          let pact = ms.pact.lock().unwrap();
+          pact.interactions().len()
+        },
+        Either::Right(ms) => ms.pact.interactions.len()
+      };
+      send_metrics(MetricEvent::ConsumerTestRun {
+        interactions,
+        test_framework: "pact_ffi".to_string(),
```

Comment:
Should these be coming from the client library, so we can disambiguate languages? I think we do this on the provider side right?

Reply:
Yes, but there is no way to get that info from within the `pactffi_cleanup_mock_server` function.

Reply:
Right, yes that makes sense. We can always add later